tests/installation/partitioning_iscsi.pm

```diff
@@ -13,14 +13,15 @@ use testapi;
 
 sub run() {
     send_key "alt-c";    # create partition setup
-    sleep 2;
+    wait_still_screen(2);
     assert_screen "preparing-disk-select-iscsi-disk";
     send_key "alt-1";    # select ISCSI disk
     send_key "alt-n";    # next
-    assert_screen "preparing-disk-install-on";
-    send_key "alt-e";    # use entire iscsi disk
-    send_key "alt-n";    # next
-    assert_screen "edit-proposal-settings";
+    if (check_screen "preparing-disk-use-entire-disk-button", 10) {
+        send_key "alt-e";    # use entire iscsi disk
+    }
+    assert_screen "preparing-disk-overview";
```

Comment:
1. on performance problems the check_screen with timeout of 10 seconds might fail, e.g. miss the screen
2. if the screen is not there we waste 10 seconds, not much, right

I won't mind merging this PR unchanged if you don't want to change